aten/src/ATen/native/native_functions.yaml

```diff
@@ -7704,8 +7704,25 @@
   python_module: linalg
   variants: function
 
+## Undocumented functions intended for internal use only
+- func: _c_style_div(Tensor self, Tensor other) -> Tensor
```

Comment:
we technically don't need the Tensor overloads, right?

Reply:
nor do we need these to be functions.

Reply:
We can prune this hidden function back to just the needed version, sure.